onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -549,6 +543,47 @@
     <string name="navigation_drawer_open">Open navigation drawer</string>
     <string name="navigation_drawer_close">Close navigation drawer</string>
 
+    <!-- Issue reporting implementation-->
+    <string name="region_dialog_Message2"> Are you in %1$s</string>
+    <string name="report_dialog_out_of_region_message"> Please select a bus stop.</string>
+    <string name="report_dialog_trip"> Please select a trip.</string>
+
+    <!-- report issue screen-->
+    <string name="ri_desc">Issue Description</string>
+    <string name="ri_button_camera">Camera</string>
+    <string name="ri_button_image">ADD IMAGE</string>
+    <string name="ri_button_gallery">Gallery</string>
+    <string name="ri_address_not_found">Address not found. Please check your address.</string>
+    <string name="ri_location_problem_info">Error getting location, please tap to map</string>
+    <string name="ri_location_problem">Error getting location</string>
+    <string name="ri_service_description_problem">Service description could not found.</string>
+    <string name="ri_service_default">Choose an issue</string>
+    <string name="ri_service_stop">Stop Problem</string>
+    <string name="ri_service_trip">Trip Problem</string>
+    <string name="ri_user_name">Name</string>
```

Comment:
I haven't looked at this in the UI yet, but should one of these be first name, and the other last name?